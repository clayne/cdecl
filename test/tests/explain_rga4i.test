cdecl @ @ @ explain register int a[4] @ 65
