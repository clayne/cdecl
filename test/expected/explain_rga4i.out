explain register int a[4]
                     ^
22: error: array can not be register
